docs/ref/item.rst

```diff
@@ -26,6 +26,14 @@ Say for instance you have 2 parameters configured: Adults (id: adults), and Chil
 
 Booking parameters have many options, and can be configured to control inventory in very specific ways.  See the Checkfront support centre or ask us for more information.
 
+Public API
+----------
+If the Public API is enabled, unauthenticated users can make requests to the */item*, */item/{id}*, */item/cal*, and */item/{id}/cal* end points. This is the same functionality as a customer viewing the item or availability calendar on the `Hosted Booking Page <https://support.checkfront.com/hc/en-us/articles/115004917593-Hosted-Booking-Page>`_.
+
+For the *item/* and *item/{id}* endpoints, the responses between the internal and public API versions have and identical structure. The public response for these endpoints will only return customer rules and item visibility; staff rules and visibility will not be returned. 
```

Comment:
Typo in `public API versions have and identical structure.`. `and` should be `an`